install.sh

```diff
@@ -143,48 +139,28 @@ if [ -d "${CREW_PREFIX}" ]; then
     echo_info "Continue?"
     select continue in "Yes" "No"; do
       if [[ "${continue}" == "Yes" ]]; then
-        # Let's see if avoiding sudo breaks anything.
-        # sudo find "${CREW_PREFIX}" -mindepth 1 -delete
-        find "${CREW_PREFIX}" -mindepth 1 -delete
+        sudo find "${CREW_PREFIX}" -mindepth 1 -delete
         break 2
       else
         exit 1
       fi
     done
   fi
 else
-  echo_error "${CREW_PREFIX} does not exist. (It should be created or mounted prior to install!)"
-  exit 1
+  mkdir -p "${CREW_PREFIX}" || sudo mkdir -p "${CREW_PREFIX}"
```

Comment:
It's fine to use `sudo mkdir -p "${CREW_PREFIX}"`.  We change the perms later anyway.